code/.repronim.bashrc

```diff
@@ -0,0 +1,99 @@
+set -o noclobber  # do not overwrite existing files
+set -o emacs  # use the emacs shortcuts
+
+# function to help remind users about the shortcuts
+print_shortcuts() {
```

Comment:
I think I like `repronim_bashrc_help` because the question mark could have [unexpected behavior](https://askubuntu.com/questions/335321/bash-function-named-with-the-question-mark-char-does-not-work-anymore)